scripts/base/frameworks/broker/store.zeek

```diff
@@ -157,7 +157,7 @@ export {
 	## Returns: the result of the query.
 	global get: function(h: opaque of Broker::Store, k: any): QueryResult;
 
-	## Insert a key-value pair in to the store, but only if the key does not
+	## Insert a key-value pair into the store, but only if the key does not
```

Comment:
`into` has tended to be the most controversial change i've offered of late. happy to drop it or anything else